.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+matrix:
```

Comment:
It's complicated to make travis work with tox. This is similar to how pip itself manages it.